### PR TITLE
Proposal: Submit Hubble.gl project to UCF

### PR DESCRIPTION
**Pull request by chrisgervang**

### CODE REVIEW

**Comment by galemccommons** [comment]
@heshan0131 can you please review before I merge? Thank you!

File: proposals/hubble-gl.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+Transferring to an open source foundation will bring in strong technical leadership to Hubble.gl and increase the appeal for high effort collaborations between itself and other vis.gl projects.
+
+## Link to current Code of Conduct
+
+- [hubble.gl](https://github.com/uber/hubble.gl/blob/master/CODE_OF_CONDUCT.md)
+
+## Sponsor from TAC
+
+Ib Green
```
Comment (heshan0131) [inline proposals/hubble-gl.md]: ib is not on TAC, please just put me

**Comment by heshan0131** [comment]
Added 1 comment

File: proposals/hubble-gl.md (hunk 1 above)
Comment (chrisgervang) [reply proposals/hubble-gl.md]: Corrected, thanks

**Review by heshan0131 — APPROVED** [review]

**Pull request merged**